src/main/java/com/example/EnumDemo/mapper/DressMapper.java

```diff
@@ -0,0 +1,43 @@
+package com.example.EnumDemo.mapper;
+
+import com.example.EnumDemo.dto.DressDto;
+import com.example.EnumDemo.dto.showDressDto;
+import com.example.EnumDemo.entity.DressEntity;
+import com.example.EnumDemo.enumC.Size;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface DressMapper {
+    @Mapping(target = "size", source = "dressEntity.size", qualifiedByName = "getDressSizeToLong")
+    DressDto entityToDto(DressEntity dressEntity);
+
+    @Mapping(target = "size", source = "dressDto.size", qualifiedByName = ("getDressSize"))
+    @Mapping(target = "brand", source = "dressDto.brand")
+    @Mapping(target = "color", source = "dressDto.color")
+    @Mapping(target = "type", source = "dressDto.type")
+    @Mapping(target = "price", source = "dressDto.price")
+    DressEntity dtoToEntity(DressDto dressDto);
+
+    @Mapping(target = "size",source = "dressDto.size")
+    @Mapping(target = "brand", source = "dressDto.brand")
+    @Mapping(target = "color", source = "dressDto.color")
+    @Mapping(target = "type", source = "dressDto.type")
+    @Mapping(target = "price", source = "dressDto.price")
+    List<showDressDto> entityToShowDressDto(List<DressEntity> dressEntity);
```

Comment:
```suggestion
    List<showDressDto> entitiesToShowDressDtos(List<DressEntity> dressEntity);
```
The name of the method should convey that it is related to list/collection

Do not add mapping to the method that returns the list. 
Instead, create a new method entityToShowDressDto that take dressEntity as parameter and return ShowDressDto. Add mapping on this method.

Reply:
Updated entityToShowDressDto for single DressEntity to ShowDressDto, and removed entitiesToShowDressDto.